web/src/app/admin/users/users-list.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container">
```

Comment:
rename component and page name to users rather than users-list

Reply:
Done